src/ex5/server/db/migrations/20220618215213-task_id_auto_increment.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        await queryInterface.changeColumn("Tasks", "id", {
+            type: Sequelize.INTEGER,
```

Comment:
Isnt it already initialized as autoIncrement in src/ex5/server/db/migrations/20220618195035-create-task.js:7?